admin-dev/themes/new-theme/js/pages/attribute-group/index.ts

```diff
@@ -53,4 +53,13 @@ $(() => {
 
   const showcaseCard = new ShowcaseCard('attributesShowcaseCard');
   showcaseCard.addExtension(new ShowcaseCardCloseExtension());
+
+  window.prestashop.component.initComponents(
+    [
+      'TranslatableInput',
+      'TranslatableField',
+    ],
+  );
+
+  new window.prestashop.component.ChoiceTree('#attribute_group_shop_association').enableAutoCheckChildren();
```

Comment:
Not blocking but could you move these selectors into a separate mapping file, please? It makes it easier to centralize them and modify them when a refacto/modification is needed